content/en/docs/setup/install/helm/index.md

```diff
@@ -151,10 +107,10 @@ installed above.
 
 1. Delete Istio base chart:
 
-    {{< warning >}}
+    {{< tip >}}
```

Comment:
We should probably move that to the top of the uninstall portion.

Reply:
Why? It seems best to keep it right next to where a user may expect them to be deleted